src/strategies/index.ts

```diff
@@ -690,7 +691,7 @@ const strategies = {
   'sunrisegaming-staking': sunriseGamingStaking,
   'single-staking-autocompound-balanceof': singleStakingAutoCompoundBalanceOf,
   'single-staking-longtermstaking-balanceof':
-    singleStakingLongTermStakingBalanceOf,
+  singleStakingLongTermStakingBalanceOf,
```

Comment:
Hi @JD0x2e Make sure there are no other changes in this file, I think it is lint issue, else everything looks good :)